src/System.Diagnostics.Process/tests/ProcessStartInfoTests.netcoreapp.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public partial class ProcessStartInfoTests
+    {
+        [Fact]
+        public void UnintializedArgumentList()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo();
+            Assert.Equal(0, psi.ArgumentList.Count);
+
+            psi = new ProcessStartInfo("filename", "-arg1 -arg2");
+            Assert.Equal(0, psi.ArgumentList.Count);
+        }
+
```

Comment:
Since we decided that `Arguments` and `ArgumentList` won't refresh each other's values, might be worth having an explicit test for that so it's clear this was a deliberate decision on our part.

Reply:
Never mind - I see this is already covered.